orchestrator-salt/src/main/resources/salt/salt/cloudera/agent/init.sls

```diff
@@ -8,7 +8,7 @@ install-cloudera-manager-agent:
 
 install-psycopg2:
   cmd.run:
-    - name: pip install --upgrade psycopg2>=2.5.4
+    - name: pip install --upgrade 'psycopg2>=2.5.4'
```

Comment:
Could you change this to use pip state instead of cmd? It should have written that way in the first place
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pip_state.html

Reply:
Tried it, but I'm getting the following error: `An importable Python 2 pip module is required but could not be found on your system.`